### DISCUSSION TITLE
History.push() not rerendering component

### DISCUSSION THREAD

**Discussion by cbjones93** [nss-day-cohort-47/C47-Discussions#86] [Help Me]
Hello all, I am attempting to use history.push() because I know window.location.reload() isn't an option anymore. Whenever I try to use it however, my card that was displaying the information simply disappears forcing me to refresh the page to get the info back, the function works just not the history.push() the way its supposed to.
Here's my code:

```const history = useHistory();
const [followers, setFollowers] = useState([]);
    const handleDeleteFollowing = id => {
        deleteFollowing(id)
            .then(() => getAllFollowing().then(setFollowing)).then(()=>history.push("/follow"));
    };

**Comment by BryanNilsen** [comment]
Sorry for the late set of eyes on this... did you find a solution?
Was there a matching Route for the "/follow" path??
